e2e/test/scenarios/admin/performance/helpers/e2e-performance-helpers.ts

```diff
@@ -60,61 +58,15 @@ export const setupDashboardTest = (
     dashboardDetails,
     visitDashboard: true,
   });
-  const reload = reloadDashboard;
+
   const waitForQuery = () => cy.wait("@dashcardQuery");
   const visitItem = () =>
     cy.then(function () {
       visitDashboard(this.dashboardId);
     });
-  return { reload, visitItem, waitForQuery };
-};
-
-export const reloadItem = ({
-  /** 'Model' in the sense of 'type of thing' */
-  model,
-  endpointAlias,
-}: {
-  model: CacheableModel;
-  endpointAlias: string;
-}) => {
-  log(`Reload ${model}`);
-  cy.reload();
-  return cy.wait(endpointAlias).then(interception => {
-    cy.wrap(interception.response?.body.running_time).as("queryRuntime");
-  });
+  return { visitItem, waitForQuery };
 };
 
-export const reloadQuestion = () =>
-  reloadItem({ model: "question", endpointAlias: "@cardQuery" });
-export const reloadDashboard = () =>
-  reloadItem({ model: "dashboard", endpointAlias: "@dashcardQuery" });
-
-export const resultIsCached = (previousResult: string, nextResult: string) =>
-  previousResult === nextResult;
-export const resultIsFresh = (previousResult: string, nextResult: string) =>
-  previousResult !== nextResult;
-
-/** The caching tests load a question (sometimes inside a dashboard) which has
- * just one result: an MD5 string. This function finds that string and wraps
- * it. */
-export const wrapResult = () =>
-  cy.url().then(url => {
-    const timeout = 10000 + questionRuntime * 3;
-    const cell = url.includes("dashboard")
-      ? cy.findAllByTestId("cell-data", { timeout }).first()
-      : cy
-          .get("#main-data-grid", { timeout })
-          .findAllByTestId("cell-data")
-          .first();
-    cell.invoke("text").then(result => {
-      expect(
-        result.length,
-        "The result, ${result}, has the correct length",
-      ).to.eq(32);
-      return cy.wrap(result);
-    });
-  });
-
```

Comment:
all of this wrapping stuff is confusing and unnecessary if we just have a known value